specifyweb/specify/models.py

```diff
@@ -7505,15 +7503,39 @@ class CollectionObjectType(models.Model):
     text3 = models.TextField(blank=True, null=True, unique=False, db_column='Text3', db_index=False)
     
     # Relationships: Many-to-One
-    collection = models.ForeignKey('Collection', db_column='CollectionID', related_name='collectionobjecttypes', null=False, on_delete=protect_with_blockers)
-    taxontreedef = models.ForeignKey('TaxonTreeDef', db_column='TaxonTreeDefID', related_name='collectionobjecttypes', null=False, on_delete=protect_with_blockers)
+    collection = models.ForeignKey('Collection', db_column='CollectionID', related_name='cotypes', null=False, on_delete=protect_with_blockers)
+    taxontreedef = models.ForeignKey('TaxonTreeDef', db_column='TaxonTreeDefID', related_name='cotypes', null=False, on_delete=protect_with_blockers)
     createdbyagent = models.ForeignKey('Agent', db_column='CreatedByAgentID', related_name='+', null=True, on_delete=protect_with_blockers)
     modifiedbyagent = models.ForeignKey('Agent', db_column='ModifiedByAgentID', related_name='+', null=True, on_delete=protect_with_blockers)
     
     class Meta:
         db_table = 'collectionobjecttype'
         ordering = ()
-        unique_together = (('collection', 'isdefault'))
+
+    timestamptracker = FieldTracker(fields=['timestampcreated', 'timestampmodified'])
+    save = partialmethod(custom_save)
+
+class CollectionObjectGroupType(models.Model):
+    specify_model = datamodel.get_table('collectionobjectgrouptype')
+
+    # ID Field
+    id = models.AutoField(primary_key=True, db_column='COGTypeID')
+
+    # Fields
+    name = models.CharField(blank=False, max_length=255, null=False, unique=False, db_column='Name', db_index=False) # microscope slide, whole rock, or piece of bark
+    cogtype = models.CharField(blank=True, max_length=255, null=False, unique=False, db_column='COGType', db_index=False) # discrete, consolidated, or drill core
```

Comment:
Can we not rename this to simply `type`? 
Or is that too ambiguous?

Reply:
I like to avoid since type is a built-in function in Python, so it's bad practice to use it as a variable name.  But it seems we have other cases already from older tables.  I'll go ahead and change it, it's probably more clear for the users.